js/script.js

```diff
@@ -55,6 +100,50 @@ for (var i = 0; i < choiceOptions.length; i++) {
 }
 
 function selectedOption(e) {
-  var userChoice = e.currentTarget.children[0].src;
-  document.getElementsByClassName("user-choice__icon")[0].src = userChoice;
+  model.userChoice = e.currentTarget.children[0].dataset.value;
+  var userChoiceUrl = e.currentTarget.children[0].src;
+  document.getElementsByClassName("user-choice__icon")[0].src = userChoiceUrl;
 }
+
+let view = {
+  displayUsername: function() {
+    var elementWithUsername = document.getElementsByClassName("jsUsername")[0];
+    elementWithUsername.innerHTML = model.username + '\'s' + ' score';
+  },
+  displayGameType: function(starsNumber) {
+    var scoreBoxes = document.getElementsByClassName("score-box");
+    for (var j = 0; j < scoreBoxes.length; j++) {
+      for (var i = 0; i < starsNumber; i++) {
+        var starIcon = document.createElement("img");
+        starIcon.src = 'img/star-24-empty.png';
+        starIcon.alt = 'star icon';
+        starIcon.classList.add("score-box__icon--star");
+        scoreBoxes[j].appendChild(starIcon);
+      }
+    }
+  },
+  displayMessage: function(message) {
+	var messageBox = document.getElementsByClassName("message-box__text")[0];
+	messageBox.innerHTML = message;
+  }
+};
+
+let controller = {
+
+};
+
+let model = {
```

Comment:
I think declarations should be at the top of a file - remember about the case when and where you can access variable declared by `const` or `let`. Here it is not the case, but remember about it